<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Организация системы мониторинга за компьютером и визуализатор полученных данных </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="widgetCPU" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetRAM" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetDisk" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetNet" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartCPU">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать мониторинг</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveCPU">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить лог в файл</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStopCPU">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Остановить монитор</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStopRAM">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Остановить монитор</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveRAM">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить лог в файл</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartRAM">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать мониторинг</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStopDisk">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Остановить монитор</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveDisk">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить лог в файл</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartDisk">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать мониторинг</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStopNet">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>550</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Остановить мониторы</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveNet">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>590</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить лог в файл</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartNet">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>510</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать мониторинг</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartAll">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Активировать все мониторы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStopAll">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Остановить все мониторы</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSravnenie">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Запустить сравнение статистики на графике</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAbout">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSysInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Информация о системе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRaspisanie">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить мониторинг по расписанию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveSWAP">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>310</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить лог в файл</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStopSWAP">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Остановить монитор</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartSWAP">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать мониторинг</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStopProcess">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Остановить монитор</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveProcess">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>180</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить лог в файл</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetProcess" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetSWAP" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartProcess">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать мониторинг</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetTCP" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>510</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonProcess">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Процессы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTCPDUMP">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>670</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>tcpdump</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSNMP">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>630</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SNMP информация</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetNet2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>421</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widgetTCP2" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>620</y>
      <width>421</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>390</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Доступные сетевые интерфейсы (по-умолчанию выбран первый):</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
